docs/design-methodology/Networking.md

```diff
@@ -487,31 +487,31 @@ An AlwaysOn application can enforce application-level network security using Net
 ### Design Considerations
 
 - AKS can be deployed in two different [networking models](https://docs.microsoft.com/azure/aks/concepts-network#azure-cni-advanced-networking).
-  - Kubenet networking: AKS nodes are integrated within an existing virtual network, but pods exist within a virtual overlay network on each node.
-  - Azure Container Networking Interface (CNI) networking: the AKS cluster is integrated within an existing virtual network resources and treated as a single large network, with pods attributed their own IP address.
+  - **Kubenet networking:** AKS nodes are integrated within an existing virtual network, but pods exist within a virtual overlay network on each node. Traffic between pods on different nodes is routed through kube-proxy.
+  - **Azure Container Networking Interface (CNI) networking:** The AKS cluster is integrated within an existing virtual network and its nodes, pods and services received IP addresses from a single large network. Different node pools can be deployed into different subnets ([in preview](https://github.com/Azure/AKS/issues/1338), February 2022).
+
+  > **Note!** Azure CNI requires, compared to Kubenet, more IP address space. Proper upfront planning and sizing of the network is required. For more information, see [Azure CNI](https://docs.microsoft.com/azure/aks/concepts-network#azure-cni-advanced-networking).
```

Comment:
all this sounds like there are only disadvantages of using CNI. Is there any doc we can link which does a deeper comparision? this one maybe? https://docs.microsoft.com/en-us/azure/aks/concepts-network#compare-network-models

Reply:
this external post also has some good points https://mehighlow.medium.com/aks-kubenet-vs-azure-cni-363298dd53bf

Reply:
This is of course not the case, the doc you've mentioned first is already linked twice. I don't wanted to repeat too much of the content that exists already on docs - for the decision process the IP demand of CNI and the conceptual difference of Kubernet (using an overlay network per node) are from my point of view the most relevant ones.

Reply:
hm fair point. I'm just saying that reading this part sounds to the reader like "why the heck would I ever bother using CNI then?"

Reply:
I'd say its mostly a networking thingy - i want my pods to have a proper IP from a range that I've defined; i want to not NAT when talking from a pod to a backend service outside of my cluster; i want my pods/services being accessible via an IP from my virtual network (which is kind of a non-argument as you'd usually not access them directly only via a service which could be using an external public or private ip address anyhow). 

Any other use cases you guys could think of? Ah yes, and windows node pool support is limited to CNI currently.

Reply:
Not saying we've to step away from CNI though - let's try to find a better way to describe that. In the end its probably more a "use CNI but do your homework upfront in regards to proper sizing"..

Reply:
For the consideration part I would keep it like this. It describes the facts and points out what is important to consider

Reply:
Updated